### ISSUE TITLE
Add stranded-ness to XS tag (refactor hardcoded logic)

### ISSUE THREAD

**Issue by angarb** [TheJacksonLaboratory/splicing-pipelines-nf#201]
## Problem
Currently, the XS awk code is hard-coded for 'first-strand' data (strType = 2). 

## Solution
We need to change this so that strType = 2 for first strand libraries and strType = 1 for second strand.

The question I asked to Alex - "If I am using the Illumina Stranded mRNA library protocol (dUTP- based), I will want to specify  strType=2 because the cDNA corresponds to strand 1 and the RNA corresponds to strand 2 (this is a fr-firststrand library). However, if I am using a fr-second-strand kit, I will want to specify strType=1 because the cDNA reflects the transcript strand?"

Alex's response - "You are correct about the  strType meaning.
In other words, strType is equal to the read whose strand agrees with the original RNA."
